docs/semgrep-code/policies.md

```diff
@@ -1,27 +1,23 @@
 ---
 slug: policies
 append_help_link: true
-title: Set rules through Policies
+title: Manage policies
 hide_title: true
-description: "The Policies page is a visual representation of the rules that Semgrep Code uses to scan code."
+description: "The Policies page is a visual representation of the rules Semgrep Code uses to scan code."
 tags:
-    - Semgrep Code
-    - Semgrep AppSec Platform
+ - Semgrep Code
+ - Semgrep AppSec Platform
 ---
 
-# Set rules through Policies
+# Manage policies
 
-Increase the breadth and depth of your scan coverage or remove noise from scans through the Policies page. Block or allow merges for pull requests (PRs) or merge requests (MRs) based on the rule that detected the finding.
+The Policies page displays the [<i class="fa-regular fa-file-lines"></i> rules](/running-rules) that Semgrep Code uses when scanning all of your repositories.
 
-The Policies page displays a visual representation of the rules that Semgrep Code uses for scanning. By default, the same rules in Policies are applied to all repositories.
+Modifications to the rules on the Policies page allow you to increase the breadth and depth of your scan coverage or remove noise from scans. You can also determine whether findings detected by a given rule can help block merges for pull requests (PRs) or merge requests (MRs).
 
-To learn more about rules, see [<i class="fa-regular fa-file-lines"></i> Running rules](/running-rules).
+## Language coverage and scan speeds
 
-:::info Speed and language coverage
-Semgrep Code identifies the languages used in your repositories and **only runs rules applicable to those languages**.
-
-For example, adding Ruby and Python rules in your Policies **does not affect the scan speed** for Python-only repositories. Only Python rules are run for Python repositories.
-:::
+Semgrep Code identifies the languages used in your repositories and only runs rules applicable to those languages. For example, adding Ruby and Python rules in your Policies doesn't affect the scan speed for Python-only repositories. Only Python rules are run for Python repositories.
 
 ## Policies page structure
 
```

Comment:
Line 24 (policies.png) - Can we update the policies screenshot :lolsob:

Reply:
Actually, I think I could do this in a separate PR (since I'm going through the image audit anyway).